packages/compartment-mapper/src/types/internal.ts

```diff
@@ -141,14 +141,15 @@ export type ChooseModuleDescriptorParams = {
    * Should be false for archives and bundles, but true for runtime.
    */
   sourceMapHook?: SourceMapHook;
-  /**
-   * Function returning a set of module names (scoped to the compartment) whose
-   * parser is not using heuristics to determine imports.
-   */
```

Comment:
Do we deem this doc redundant? Are IDEs passing thru to the doc on StrictlyRequiredFn?

Reply:
We can either keep the description in two places or not.  I figured we probably shouldn't.  This description will not be shown in the context of the options bag, no; only when you view info for `StrictlyRequiredFn`.

Reply:
I don't think there's any way to reference the `StrictlyRequiredFn` description other than using the `@inheritDoc` tag for TypeDoc; the TS language server does not understand this tag.

Reply:
...and given this is an internal type, we won't get any TypeDoc output for it _anyway_, so I think it's moot.